### ISSUE TITLE
Your build is failing

### ISSUE THREAD

**Issue by aeslinger0** [SolrNet/SolrNet#275]
There's an icon on https://github.com/SolrNet/SolrNet/blob/master/Documentation/README.md that indicates the build failed. If you follow the link to https://ci.appveyor.com/project/mausch/solrnet it appears that it's been failing for about a month. Thought you'd like to know.

**Comment by xavier-morera** [comment]
Thanks, I am trying to catch up on all the things I have to do. I know it is a slow start but I got access right about the time I was finishing up on another project that is consuming my time.

I believe next week I will get to two pull requests, modifying the contribute page and the build.

Thanks for letting me know and I apologize for those who want SolrNet to move faster.

**Comment by xavier-morera** [comment]
Hi, I got to this. Actually build is not "really" failing and tests are passing. The issue is that Gallio has an [incompatibility in the PDBs](https://github.com/Gallio/mbunit-v3/issues/6)  with Visual Studio 2015+. It works fine with Visual Studio 2013, I just checked and all tests pass. 

So for upgrading Visual Studio we would need to find another option as Gallio is not being actively developed. If anyone has a recommendation, we can talk about it. I like XUnit, and I have asked for recommendations. I will update soon on the status.

Closing for now.
